### PR TITLE
Basic table rendering

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

File: addon/templates/components/cf-field/input/table.hbs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    </tbody>
+    <tfoot>
+      <tr>
+        <td colspan={{add (count-keys questions) 1}} class="uk-text-center">
+          {{#uk-button size="small" on-click=(perform addRow)}}{{uk-icon "plus"}}{{/uk-button}}
+        </td>
+      </tr>
+    </tfoot>
+  </table>
+  {{#uk-modal visible=showModal}}
```
[the remark this answers is not in the document]
Comment (czosel) [reply addon/templates/components/cf-field/input/table.hbs]: I actually like the idea! What message were you thinking of?

File: translations/de-de.yaml [hunk 2]
```diff
@@ -39,8 +40,7 @@ caluma:
 
     form:
       name: "Name"
-      slug: "Slug"
-      description: "Beschreibung"
+      slug: "Slug" description: "Beschreibung"
```
[the remark this answers is not in the document]
Comment (czosel) [reply translations/de-de.yaml]: Good catch :)

File: addon/templates/components/cf-field/input/table.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (czosel) [reply addon/templates/components/cf-field/input/table.hbs]: Fixed

[the remark this answers is not in the document]
Comment (fkm) [reply addon/templates/components/cf-field/input/table.hbs]: Something along the line of "No document selected".

[the remark this answers is not in the document]
Comment (czosel) [reply addon/templates/components/cf-field/input/table.hbs]: Ah right - then the modal shouldn't be displayed though, so your initial suggestion is good :+1:

**Pull request merged**